src/clj/game/cards/assets.clj

```diff
@@ -1434,7 +1437,7 @@
                                           (rez-cost-bonus state side (* ap -2))
                                           (rez state side target {:no-warning true})
                                           (if (< cnt 3) (continue-ability state side (rez-ice (inc cnt)) card nil)
```

Comment:
I'd probably like the first if statement to be on a new line as well, looks better I think.